tests/app/confirmation_page/test_confirmation_page.py

```diff
@@ -15,7 +15,7 @@ def test_get_next_location_confirmation(self):
         answer_store = AnswerStore()
         answer_store.add(answer)
 
-        navigator = Navigator(load_schema_file("0_rogue_one.json"), answer_store)
+        navigator = Navigator(load_schema_file("0_rogue_one.json"), {}, answer_store)
```

Comment:
I'd agree with @jonnyshaw89 here. Looks like metadata is a fundamental component in the navigator in which it wouldn't function properly without so it should be required?